### DISCUSSION TITLE
Using htmx without jsx on Cloudflare Workers

### DISCUSSION THREAD

**Discussion by LM3Any** [honojs/hono#2768] [Q&A]
Hello and thanks in advance for your help.

Is it possible to use htmx without using jsx on Cloudflare Workers?

All the discussions, docs and examples that I found seem to require jsx but I don't want to use nodejs with Workers.

Would you please point me to a resource that explains or demos how to do this?

**Comment by NicoPlyley** [comment]
Here is an example from the creator of Hono 

https://github.com/yusukebe/hono-htmx

It uses JSX to compile the code, however, using JSX does not mean you have to use node. Node is not compatible on CF Workers/Pages anyways.

JSX is just the way for the Htmx code to be understood
